doc/examples/text/lib/main.dart

```diff
@@ -34,27 +35,27 @@ class MyTextBox extends TextBoxComponent {
 }
 
 class MyGame extends BaseGame {
-  MyGame(Size screenSize) {
+  MyGame(Vector2 screenSize) {
     size = screenSize;
     add(TextComponent('Hello, Flame', config: regular)
       ..anchor = Anchor.topCenter
-      ..x = size.width / 2
+      ..x = size.x / 2
       ..y = 32.0);
 
     add(TextComponent('center', config: tiny)
       ..anchor = Anchor.center
```

Comment:
this could be setByPosition(size / 2) :)

Reply:
True, much nicer. Fixed!